specification/storagecache/resource-manager/readme.ruby.md

```diff
@@ -12,11 +12,22 @@ azure-arm: true
 
 ``` yaml $(ruby) && $(multiapi)
 batch:
+  - tag: package-2020-10-01
   - tag: package-2020-03-01
   - tag: package-2019-11-01
   - tag: package-2019-08
 ```
 
+### Tag: package-2020-10-01 and ruby
+
+These settings apply only when `--tag=package-2020-10-01 --ruby` is specified on the command line.
+Please also specify `--ruby-sdks-folder=<path to the root directory of your azure-sdk-for-ruby clone>`.
+
+``` yaml $(tag) == 'package-2020-10-01' && $(ruby)
```

Comment:
@romahamu , Does the space before "yaml" here affect SDK generation? Not sure if that space is OK in the language-specific readmes (as opposed to overall readme comment above).

Reply:
Not seeing what needs to be fixed in these readmes.  The spacing matches everything that is already there.